app/javascript/packs/intlTelInput.js

```diff
@@ -0,0 +1,40 @@
+import intlTelInput from "intl-tel-input";
+
+import "intl-tel-input/build/css/intlTelInput.css";
+import "intl-tel-input/build/js/utils.js";
+
+// Set up all phone number inputs with intl-tel-input. We will add a hidden field
+// that allows forms to submit the full number. We use HTML5 validation.
+$(document).ready(() => {
```

Comment:
This is our code. We can write ES6 and webpack/Babel will transpile it to what's supported by our target browsers. We should be able to use TypeScript as well (my preferred option), but one step at a time.

Reply:
Makes sense - thanks!